lib/modules/datasource/orb/index.spec.ts

```diff
@@ -92,5 +92,19 @@ describe('modules/datasource/orb/index', () => {
       expect(res).toMatchSnapshot();
       expect(res?.homepage).toBe('https://google.com');
     });
+
+    it('supports other registries', async () => {
+      httpMock
+        .scope('https://cci.internal.dev')
+        .post('/graphql-unstable')
+        .reply(200, orbData);
+      const res = await getPkgReleases({
+        datasource,
+        packageName: 'hyper-expanse/library-release-workflows',
+        registryUrls: ['https://cci.internal.dev'],
+      });
+      expect(res).toMatchSnapshot();
```

Comment:
please use to equal or to match object. read our dev guides

Reply:
I was just pattern matching to the rest of this test file. Happy to update it.